spec/support/jasmine-browser.json

```diff
@@ -8,6 +8,7 @@
     "**/*[sS]pec.js"
   ],
   "helpers": [
+    "https://unpkg.com/jasmine-ajax@4.0.0/lib/mock-ajax.js",
```

Comment:
Should we vendor this?

Reply:
We've been experimenting with the CDN approach on Content Publisher before: https://github.com/alphagov/content-publisher/blob/3f14860c12aae112f0da2afb5314740d253177d2/spec/support/jasmine-browser.json 

It seems a bit cleaner than vendoring.

Ideally we'd pull these through with an NPM module but we don't know how to do that yet.

Would some more info in the commit message help?

Reply:
It's fine 👍